### PR TITLE
Update network-exchange-clc-colocation-endpoint-guide.md

### PR DESCRIPTION
**Pull request by marcopao**
Added information for migration of existing CLC accounts and note pointing readers to the Network Exchange Architecture and Technical Guide.

### CODE REVIEW

File: Network/network-exchange-clc-colocation-endpoint-guide.md [hunk 1]
```diff
@@ -21,11 +23,12 @@
...
   * QFX-SFP-10GE-LR SFP+ 10GBASE-LR 10 Gigabit Ethernet Optics, 1,310 nm for 10 km transmission on single mode fiber-optic (SMF)
   * CSCO-GLC-T 1000BASE-T SFP Copper SFP, Up to 100M (328 ft.) on Cat 5 unshielded twisted-pair copper cabling
 * The user will obtain the proper optics for their end of the cross connect, matching speed and type (SMF or Cat 5). 
 * The Network Exchange fabric leverages the economics of shared networking while logically isolating network traffic between Exchanges, even within the same End User account. Dedicated access to CenturyLink Cloud is not supported with Network Exchange.
 * Currently, an End User may only add the colocation endpoint(s) present in the same metropolitan area as the serving instance of Network Exchange. Where more than one colocation endpoint is served, all may be included in a given Exchange. See the *Network Exchange Availability Matrix and Configuration Guide* for available endpoints per metro area. 
+* For additional information regarding the Network Exchange architecture and use cases, please consult the *Network Exchange Architecture and Technical Guide*.
```
Comment (chrislittle) [inline Network/network-exchange-clc-colocation-endpoint-guide.md]: can we provide a link to this? or is it somewhere hidden?

**Comment by marcopao** [comment]
Chris,


Thanks for asking.  Previous author/PO had not included links, so I wasn't sure that linking between multiple docs was permitted.


I can certainly add those.

Marco

________________________________

**Comment by jim-phillips** [comment]
@marcopao please update by the end of the week

**Comment by jim-phillips** [comment]
@marcopao last chance to fix this.  I will close this due to inactivity on 4/11.

**Pull request closed without merging**